### PR TITLE
Updated APM docs

### PR DESCRIPTION
**Pull request by paperclypse**
This PR is ready for review now. Do not merge in until tomorrow, April 1.

### CODE REVIEW

File: src/content/docs/agents/java-agent/getting-started/introduction-new-relic-java.mdx [hunk 1]
```diff
@@ -49,6 +49,38 @@ To [view your app's performance in the New Relic UI](/docs/apm/applications-menu
 
 You can also [extend your agent instrumentation](#extend-instrumentation) and explore your data with other New Relic products, including New Relic One's [data explorer](/docs/query-your-data/explore-query-data/data-explorer/introduction-data-explorer) and [dashboards](/docs/query-your-data/explore-query-data/dashboards/introduction-new-relic-one-dashboards).
 
+## Evaluate our Java agent with guided install [#guided-install]
+
+Our guided install creates a customized CLI command for your environment that downloads and installs the Java agent. 
+
+<Callout variant="caution">
+The guided install version of the Java agent is for evaluation purposes only. It will not persist when you restart your Java application.
```
Comment (Broseberg) [inline src/content/docs/agents/java-agent/getting-started/introduction-new-relic-java.mdx:57]: Can we had some friendlier language than "for evaluation purposes only"? That seems like it's discouraging me from using it. Maybe something like, "The guided install lets you test out data with our Java agent, without retaining any permanent data. The data will not remain when you restart your Java application."?

File: src/content/docs/agents/java-agent/getting-started/introduction-new-relic-java.mdx [hunk 2]
```diff
@@ -49,6 +49,38 @@ To [view your app's performance in the New Relic UI](/docs/apm/applications-menu
...
+</Callout> 
+
+![A screenshot of the guided install CLI.](./images/guided-install-cli.png "The guided install CLI.")
+
+Ready to get started? Click one of these buttons to try it out.
```
Comment (Broseberg) [inline src/content/docs/agents/java-agent/getting-started/introduction-new-relic-java.mdx:62]: "click one" makes it seem like anyone could do either, which isn't really the case. Maybe change this to, "Ready to get started? Click the Guided install button. If your account reports data through our [EU datacenter](link to datacenter doc), Click on the EU Guided install."

File: src/content/docs/agents/java-agent/getting-started/introduction-new-relic-java.mdx [hunk 3]
```diff
@@ -49,6 +49,38 @@ To [view your app's performance in the New Relic UI](/docs/apm/applications-menu
...
+<ButtonLink
+  role="button"
+  to="https://one.eu.newrelic.com/launcher/nr1-core.home?pane=eyJuZXJkbGV0SWQiOiJucjEtY29yZS5ob21lLXNjcmVlbiJ9&cards[0]=eyJuZXJkbGV0SWQiOiJucjEtaW5zdGFsbC1uZXdyZWxpYy5ucjEtaW5zdGFsbC1uZXdyZWxpYyIsInBhdGgiOiJhcG0iLCJsYW5ndWFnZSI6ICJqYXZhIiwgInJlY2lwZU5hbWUiOiAiamF2YS1hZ2VudC1pbnN0YWxsZXIiLCAiaW5pdGlhbEFjdGlvbkluZGV4IjoyLCJhY2NvdW50SWQiOm51bGx9"
+  variant="primary"
+>
+  EU Guided install
+</ButtonLink>
+</ButtonGroup>
+
+In addition to the Java agent, you can use this to install our infrastructure agent to discover other applications, infrastructure, and log sources running in your environment and recommend which ones should be instrumented. The install automates the configuration and deployment of each system you choose to instrument.
```
Comment (Broseberg) [inline src/content/docs/agents/java-agent/getting-started/introduction-new-relic-java.mdx:82]: So the Java agent guided install doesn't completely install the Java agent, but clicking those buttons will still let me install Infra and other things? (Not necessarily a critique of the writing here, just making sure I understand what all can happen when I click the button.)

File: src/content/docs/agents/net-agent/getting-started/introduction-new-relic-net.mdx [hunk 4]
```diff
@@ -39,6 +39,32 @@ With New Relic's support for .NET, you can monitor your apps in dynamic or distr
 * Self-hosted Windows and Linux systems
 * Amazon AWS EC2 VMs
 
+## Install the agent on Windows with our guided install [#guided-install]
+
+Our guided install creates a customized CLI command for your environment that downloads and installs the .NET agent for Windows IIS applications. 
+
+Ready to get started? Click one of these buttons to try it out.
+
```
Comment (Broseberg) [inline src/content/docs/agents/net-agent/getting-started/introduction-new-relic-net.mdx:47]: Ditto for the "Click one" comment I had for Java, I wonder if there's clearer language we can use about which button.

File: src/content/docs/agents/java-agent/getting-started/introduction-new-relic-java.mdx (hunk 1 above)
Comment (paperclypse) [reply src/content/docs/agents/java-agent/getting-started/introduction-new-relic-java.mdx:57]: I agree, your language is a lot friendlier :)

File: src/content/docs/agents/java-agent/getting-started/introduction-new-relic-java.mdx (hunk 2 above)
Comment (paperclypse) [reply src/content/docs/agents/java-agent/getting-started/introduction-new-relic-java.mdx:62]: Great suggestion!

File: src/content/docs/agents/java-agent/getting-started/introduction-new-relic-java.mdx (hunk 3 above)
Comment (paperclypse) [reply src/content/docs/agents/java-agent/getting-started/introduction-new-relic-java.mdx:82]: Well, it does completely install the Java agent. It's just that the Java agent installation isn't persistent. When you restart your Java application, it effectively breaks your application's connection with the agent. This means that you'll no longer be sending data to New Relic until you restart the agent too.

File: src/content/docs/agents/net-agent/getting-started/introduction-new-relic-net.mdx (hunk 4 above)
Comment (paperclypse) [reply src/content/docs/agents/net-agent/getting-started/introduction-new-relic-net.mdx:47]: I'll think on it.

**Review by Broseberg — APPROVED** [review]

**Comment by paperclypse** [comment]
We'll need to hold this PR until Monday, April 5.

**Pull request merged**